mteb/models/llm2vec_models.py

```diff
@@ -199,6 +262,15 @@ def loader_inner(**kwargs: Any) -> Encoder:
     name="McGill-NLP/LLM2Vec-Sheared-LLaMA-mntp-unsup-simcse",
     languages=["eng_Latn"],
     open_source=True,
-    revision=None,
+    revision="a5943d406c6b016fef3f07906aac183cf1a0b47d",
     release_date="2024-04-09",
+    n_parameters=7_111_000_000,
```

Comment:
Maybe write in millions parameters?

Reply:
I think this is better, honestly. I would rather have larger numbers than introduce confusion

Reply:
yea did consider it as well. I agree with @x-tabdeveloping, but it is a minor preference.